web/cobrands/sass/_base.scss

```diff
@@ -1496,6 +1497,48 @@ html.js #map .noscript {
       background-image:url($image-sprite);
       background-position: flip(right, -341px) -3936px;
     }
+    &.expand {
+      padding: 0.6em 1em 0.5em 1em;
+      &:after {
```

Comment:
Was wrestling with this for a while this morning (my CSS-fu is weak), and @zarino suggested doing it this way, as positioning relative to the right hand side is tricky to do with `background-position`. I think it's easier with the others as they use a sprite, but I decided to use an SVG as we seem to be moving in that direction for other stuff.